src/gobapi/storage.py

```diff
@@ -218,6 +223,42 @@ def convert(entity):
     return convert
 
 
+def _add_resolve_attrs_to_columns(columns):
+    """Adds attributes to columns necessary to resolve model attributes from a view.
+
+    Looks for attributes of the form brk:sjt:heeft_bsn_voor, and will try to find the attribute heeft_bsn_voor in the
+    catalog brk and collection sjt.
+
+    Adds attribute, authority and public_name (heeft_bsn_voor in this case) to columns matching the pattern above.
+    """
+    resolve_attr_pattern = re.compile("^(\w+):(\w+):(\w+)$")
+    authorities = defaultdict(dict)
+
+    for column in columns:
+        match = re.match(resolve_attr_pattern, column.name)
+        if not match:
+            continue
+
+        catalog_abbreviation, collection_abbreviation, attribute_name = match.groups()
+
+        try:
+            catalog, collection = GOBModel().get_catalog_collection_from_abbr(catalog_abbreviation,
+                                                                              collection_abbreviation)
+            attribute = collection['attributes'][attribute_name]
+        except (NotInModelException, KeyError):
+            continue
+
```

Comment:
You can limit the use of Authority in storage by setting just the catalog, collection and attribute name.
All knowledge about an authority is then concentrated in get_value